provisioning/provisioning-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/provisioning/device/internal/contract/UrlPathBuilderTest.java

```diff
@@ -27,7 +27,7 @@
     private static final String TEST_HOST_NAME = "testHostName";
     private static final String TEST_REGISTRATION_ID = "testRegistrationId";
     private static final String TEST_OPERATION_ID = "testOperationId";
-    private static final String SERVICE_API_VERSION = "2017-11-15";
+    private static final String SERVICE_API_VERSION = "2018-09-01-preview";
```

Comment:
Shouldn't this be 2018-11-01? Here and everywhere

Reply:
Changed it in code, forgot to change it here. Wonder how this test passed